CANscope/CZUL/CZUL.ese

```diff
@@ -252,7 +252,7 @@ CYQB ATIS:-:128.300:QS:-:CYQB:ATIS:CYQB:ATIS::
 ; TCA D'OTTAWA - OTTAWA TCA (3)
 ;
 Ottawa Terminal:Terminal de OTTAWA Terminal:135.150:OA:A:CYOW:APP:CYOW:APP:0300:0337:N045.19.21.000:W075.40.09.001
-Ottawa Departures:Departs de OTTAWA Departures:128.170:OD:D:CYOW:DEP:CYOW:DEP:0300:0337:N045.19.21.000:W075.40.09.001
+Ottawa Departures:Departs de OTTAWA Departures:128.175:OD:D:CYOW:DEP:CYOW:DEP:0300:0337:N045.19.21.000:W075.40.09.001
```

Comment:
No reason to change to .175, we are sure that .170 will work.

Reply:
Agree, because I was just made aware by Vatsim not to use realistic 25 kHz frequencies to keep support for Squawkbox and FSinn pilot clients. Will change to standard 25 kHz frequencies in the future.